Mathlib/Computability/DFA.lean

```diff
@@ -28,11 +28,9 @@ Currently, there are two disjoint sets of simp lemmas: one for `DFA.eval`, and a
 - Should `mem_accepts` and `mem_acceptsFrom` be marked `@[simp]`?
 -/
 
-
-open Computability
-
 universe u v
 
+open Computability
```

Comment:
Actually the standard is to have
```
open ...
open scoped ...

universe ...
variable ...
```
It's important to have `open ...`/`open scoped ...` before `variable ...` because the latter might use some names or notations only accessible in a namespace, and `universe` is a very close cousin of `variable` so should stay next to it

Reply:
OK, then it should probably make sense to replace
```
def Language.IsRegular {T : Type u}
```
by
```
def Language.IsRegular {T : Type*}
```
right?

Reply:
However, I am not sure the existentially-quantified type will accept `Type*` in place of `Type v`.

Reply:
All I'm saying is that you should revert the change I've commented on

Reply:
Did you take into account that the universes are used outside of the section as well as inside it?